onnxruntime/python/tools/transformers/models/t5/convert_to_onnx.py

```diff
@@ -217,6 +228,7 @@ def export_onnx_models(
 
 def main():
```

Comment:
maybe comment/assert somewhere that onnx>=1.12 is needed?

Reply:
Sure. Let me add a check of ONNX version in next PR.